.scaffold/tests/bats/deployment-webhook.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+#
+# Test for webhook deployments.
+#
+# shellcheck disable=SC2030,SC2031,SC2129,SC2155
+
+load _helper.bash
+load _helper.deployment.bash
+
+@test "Missing variable checks" {
+  pushd "${LOCAL_REPO_DIR}" >/dev/null || exit 1
+  unset DREVOPS_DEPLOY_WEBHOOK_URL
+  unset DREVOPS_DEPLOY_WEBHOOK_METHOD
+  unset DREVOPS_DEPLOY_WEBHOOK_RESPONSE_STATUS
+  run scripts/drevops/deploy-webhook.sh
+  assert_failure
+  assert_output_contains "Missing required value for DREVOPS_DEPLOY_WEBHOOK_URL."
+  mock_curl=$(mock_command "curl")
+  mock_set_output "${mock_curl}" "200" 1
+  export DREVOPS_DEPLOY_WEBHOOK_URL="https://example.com"
+  unset DREVOPS_DEPLOY_WEBHOOK_METHOD
+  unset DREVOPS_DEPLOY_WEBHOOK_RESPONSE_STATUS
+  run scripts/drevops/deploy-webhook.sh
+  assert_success
+  assert_output_not_contains "Missing required value for DREVOPS_DEPLOY_WEBHOOK_METHOD."
```

Comment:
Currently scripts have default values for these two environment variables and so check will currently not fail.